src/main/java/de/bonndan/nivio/output/map/svg/SVGDocument.java

```diff
@@ -120,16 +123,14 @@ public DomContent render() {
         }).collect(Collectors.toList());
 
         //render background hexes
-        var i = 0;
-        for (int q = minQ.get(); q <= maxQ.get(); q++) {
-            for (int r = minR.get() - i; r <= (maxR.get() + (maxQ.get())); r++) {
-                background.add(new SVGHex(new Hex(q, r), "none", "#cccccc").render());
-            }
-            i++;
-        }
-
-        int paddingTopLeft = 2 * Hex.HEX_SIZE;
-
```

Comment:
Is this the logic we now have in `SVGBackgroundFactory`?

Reply:
Yes